homeassistant/components/dynalite/__init__.py

```diff
@@ -52,10 +49,6 @@
         # Store config in hass.data so the config entry can find it
         hass.data[DOMAIN][DATA_CONFIGS][host] = bridge_conf
```

Comment:
The bridge config schema should be defined in home assistant and not imported from the library and directly used. We should decouple the two APIs, dynalite from home assistant. Please define the bridge validation schema in home assistant.

Reply:
ok. will take some work but should be able to get it done later tonight

Reply:
done.